dbt/include/sqlserver/macros/adapters.sql

```diff
@@ -145,18 +132,18 @@
    type='view')-%}
    {%- set temp_view_sql = sql.replace("'", "''") -%}
 
-   {{ sqlserver__drop_relation_script(tmp_relation) }}
+   {{ sqlserver__drop_relation(tmp_relation) }}
```

Comment:
I believe this, and the other instances below, can just be `{{ adapter.drop_relation(tmp_relation) }}`

Reply:
good catch! qq: why `adapter.drop_relation()`  and not just `drop_relation()`? is it to make sure the macro can't be overwritten by a macro in the project that is also  named `drop_relation`?